neofetch

```diff
@@ -7740,6 +7740,31 @@ EOF
         "Linux Mint"* | "LinuxMint"*)
             set_colors 2 7
             read -rd '' ascii_data <<'EOF'
+${c2}             ...-:::::-... 
+${c2}          .-MMMMMMMMMMMMMMM-.
+      .-MMMM${c1}`..-:::::::-..`${c2}MMMM-.   
+    .:MMMM${c1}.:MMMMMMMMMMMMMMM:.${c2}MMMM:.
+   -MMM${c1}-M---MMMMMMMMMMMMMMMMMMM.${c2}MMM-
+ `:MMM${c1}:MM`  :MMMM:....::-...-MMMM:${c2}MMM:`
+ :MMM${c1}:MMM`  :MM:`  ``    ``  `:MMM:${c2}MMM:
+.MMM${c1}.MMMM`  :MM.  -MM.  .MM-  `MMMM.${c2}MMM.
+:MMM${c1}:MMMM`  :MM.  -MM-  .MM:  `MMMM-${c2}MMM:
+:MMM${c1}:MMMM`  :MM.  -MM-  .MM:  `MMMM:${c2}MMM:
+:MMM${c1}:MMMM`  :MM.  -MM-  .MM:  `MMMM-${c2}MMM:
+.MMM${c1}.MMMM`  :MM:--:MM:--:MM:  `MMMM.${c2}MMM.
+ :MMM${c1}:MMM-  `-MMMMMMMMMMMM-`  -MMM-${c2}MMM:
+  :MMM${c1}:MMM:`                `:MMM:${c2}MMM:
+   .MMM${c1}.MMMM:--------------:MMMM.${c2}MMM.
+     '-MMMM${c1}.-MMMMMMMMMMMMMMM-.${c2}MMMM-'
+       '.-MMMM${c1}``--:::::--``${c2}MMMM-.'
+${c2}            '-MMMMMMMMMMMMM-'
+${c2}               ``-:::::-``
+EOF
+        ;;
+
+        "Linux Mint_old")
```

Comment:
```sh
"Linux Mint Old"* | "LinuxMintOld"* | "mint_old"*)
```

Reply:
Do I need to add `"Linux Mint Old"* | "LinuxMintOld"* | "mint_old"*)` to line 7765?

Reply:
Replace the line with that, yeah.